reporters/index.js

```diff
@@ -131,14 +122,23 @@ async function run(
     periodLengthSeconds
   );
 
+  // 7. Sponsor reporter to generate metrics on monitored positions.
+  const sponsorReporter = new SponsorReporter(
```

Comment:
Just curious -- why did this move?

Reply:
@allisonlu22 requested this change. Since the sponsor table can get really long, I agree it improves readability to show the summary report first, and let the indeterminate length of the sponsor table conclude the report

Reply:
SGTM